### PR TITLE
Updated JSDocs with @throws parameter

### PR DESCRIPTION
**Pull request by Coenego**

### CODE REVIEW

**Comment by Coenego** [comment]
Assigning to @bp323 for review

**Comment by bp323** [comment]
Could you have a look at the merge conflict?

File: shared/gh/js/controllers/gh.api.event.js [hunk 1]
```diff
@@ -59,6 +60,7 @@ define(['exports'], function(exports) {
      * @param  {Function}    callback            Standard callback function
      * @param  {Error}       callback.err        Error object containing the error code and error message
      * @param  {Event}       callback.reponse    The created event
+     * @throws {Error}                           A parameter validation error
      *
```
Comment (bp323) [inline shared/gh/js/controllers/gh.api.event.js]: Extra line can go

**Comment by bp323** [comment]
Just one comment left + merge conflict

**Comment by Coenego** [comment]
Reassigning to @bp323

**Comment by bp323** [comment]
Verified and merged.

**Pull request merged**